eng/Packages.Data.props

```diff
@@ -100,7 +100,9 @@
   -->
   <ItemGroup Condition="'$(IsClientLibrary)' == 'true' and $(MSBuildProjectName.StartsWith('Microsoft.'))">
     <PackageReference Update="CloudNative.CloudEvents" Version="2.0.0-beta.1" />
-    <PackageReference Update="Microsoft.Azure.WebJobs" Version="3.0.25" />
+    <PackageReference Update="Microsoft.Azure.WebJobs" Version="5.0.0" />
```

Comment:
do we need to bump this to 5.0 ? how's that bump related to extension and providers below?
(by looking at rest of the pr I'd guess we just need providers but might be I'm missing something).

Reply:
Microsoft.Azure.WebJobs is used by the new storage providers, and it was edited so it will need a version bump at the very least. We'll need to decide the version.